hphp/runtime/ext/std/ext_std_math.cpp

```diff
@@ -495,6 +495,16 @@ int64_t HHVM_FUNCTION(mt_rand,
 
 double HHVM_FUNCTION(lcg_value) { return math_combined_lcg();}
 
+Variant HHVM_FUNCTION(intdiv, int64_t numerator, int64_t divisor) {
+  if (divisor == 0) {
+    raise_warning("intdiv(): Division by zero");
+    return false;
+  } else if (divisor == -1 && numerator == LONG_MIN) {
```

Comment:
Instead of `LONG_MIN` it would be better to use the `int64_t` min here (which do not match e.g. on windows). I think in C++ you get this via numeric_limits.